docs/agent-k8s-install-helm.md

```diff
@@ -0,0 +1,117 @@
+# Install Using Helm
+
+Use the Helm package manager to install the Smart Agent to Kubernetes
+environments.
+
+If you don't have Helm installed already, use kubectl
+to install the Smart Agent. To learn more, see
+[Install the Smart Agent using kubectl](agent-k8s-install-kubectl.md).
+
+## Prerequisites
+
+* Linux kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
+* Terminal or a similar command-line interface application
```

Comment:
This is kind of a given if you are using Helm.

Reply:
Should I remove it?